iaso/tests/api/test_permissions.py

```diff
@@ -0,0 +1,61 @@
+from hat import settings
+from hat.menupermissions.constants import PERMISSIONS_PRESENTATION
+from hat.menupermissions.models import CustomPermissionSupport
+from iaso.test import APITestCase
+from iaso import models as m
+from django.contrib.auth.models import Permission
+
+from iaso.utils.module_permissions import account_module_permissions
+
+
+class OrgUnitAPITestCase(APITestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.star_wars = star_wars = m.Account.objects.create(name="Star Wars")
+        cls.yoda = cls.create_user_with_profile(
+            username="yoda", account=star_wars, permissions=["iaso_users", "iaso_user_roles"]
+        )
+
+        cls.MODULES = [
+            {"name": "Default", "codename": "DEFAULT"},
+        ]
+        cls.star_wars.modules = [module["codename"] for module in cls.MODULES]
+        cls.star_wars.save()
+
+    def test_permission_list_without_auth(self):
+        """GET /permissions/ without auth should result in a 401"""
+
+        response = self.client.get("/api/permissions/")
+        self.assertJSONResponse(response, 401)
+
+        response = self.client.get("/api/permissions/grouped_permissions/")
+        self.assertJSONResponse(response, 401)
+
+    def test_get_all_permissions_without_grouping(self):
+        """Get /permissions all permissions without grouping them"""
+
+        self.client.force_authenticate(self.yoda)
+
+        response = self.client.get("/api/permissions/")
+        self.assertJSONResponse(response, 200)
+        account = self.yoda.iaso_profile.account
+        account_modules = account.modules if account.modules else []
```

Comment:
```suggestion
        account_modules = account.modules
```

This should never be an empty `[]` because it's explicitely defined in `setUpTestData`.

You could use `account.modules` directly without an additional variable.